packages/ui/src/views/repo/components/summary/summary.tsx

```diff
@@ -0,0 +1,128 @@
+import { useNavigate } from 'react-router-dom'
+
+import {
+  Avatar,
+  AvatarFallback,
+  AvatarImage,
+  ButtonGroup,
+  CommitCopyActions,
+  Icon,
+  Spacer,
+  StackedList,
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableHeader,
+  TableRow,
+  Text
+} from '@/components'
+import { LatestFileTypes, RepoFile, SummaryItemType } from '@/views'
+import { getInitials } from '@utils/stringUtils'
+
+interface SummaryProps {
+  latestFile: LatestFileTypes
+  files: RepoFile[]
+}
+
+export const TopTitle = ({ file }: { file: LatestFileTypes }) => {
+  const { user, lastCommitMessage } = file
+
+  return (
+    <ButtonGroup.Root verticalAlign="center" spacing="2">
+      <Avatar size="6">
+        <AvatarImage src={user?.avatarUrl || ''} />
+        <AvatarFallback>
+          <Text size={0} color="tertiaryBackground">
+            {getInitials(user?.name || '')}
+          </Text>
+        </AvatarFallback>
+      </Avatar>
+      <Text size={2} weight="normal" color="tertiaryBackground" wrap="nowrap">
+        {user?.name}
+      </Text>
+      <Text size={2} weight="normal" color="primary" className="line-clamp-1">
+        {lastCommitMessage}
+      </Text>
+    </ButtonGroup.Root>
+  )
+}
+
+export const TopDetails = ({ file }: { file: LatestFileTypes }) => {
+  const { sha, timestamp } = file
+  return (
+    <ButtonGroup.Root verticalAlign="center" spacing="2">
+      <CommitCopyActions sha={sha || ''} />
+      <Text className="text-borders-2" size={2} weight="normal">
+        |
+      </Text>
+      <Text size={2} weight="normal" color="tertiaryBackground">
+        {timestamp}
+      </Text>
+    </ButtonGroup.Root>
+  )
+}
+
+export const Summary = ({ latestFile, files }: SummaryProps) => {
+  const navigate = useNavigate()
+
+  return (
+    <>
+      <StackedList.Root>
+        <StackedList.Item disableHover isHeader className="px-3 py-2">
+          {latestFile ? (
+            <>
+              <StackedList.Field title={<TopTitle file={latestFile} />} />
+              <StackedList.Field right title={<TopDetails file={latestFile} />} />
+            </>
+          ) : (
+            <Text>No files available</Text>
+          )}
+        </StackedList.Item>
+      </StackedList.Root>
+      <Spacer size={4} />
+      <Table variant="asStackedList">
+        <TableHeader>
+          <TableRow>
+            <TableHead>Name</TableHead>
+            <TableHead>Last commit message</TableHead>
+            <TableHead className="text-right">Date</TableHead>
+          </TableRow>
+        </TableHeader>
+        <TableBody>
+          {files.map(file => (
+            <TableRow key={file.id} onClick={() => navigate(file.path)}>
+              <TableCell>
+                <ButtonGroup.Root
```

Comment:
Something we can look at later, but it looks like we're using a `ButtonGroup` as a layout component for non-buttons. Might be something we can swap to be just a `div` or something. Not a blocker, just something to think about.

Reply:
I have removed ButtonGroup.